install-odoo-linux-server.md

```diff
@@ -250,8 +257,3 @@ b) redémarrer odoo
 
     # /etc/init.d/odoo restart
 
-
-
--- 
-Thibault François 

```

Comment:
Maybe we should leave his name in an author section ?

Reply:
I put it at the top already :-)